HW25/index.js

```diff
@@ -0,0 +1,27 @@
+let users = [
+  { name: "John", surname: "Black", age: 18 },
+  { name: "Tom", surname: "White", age: 12 },
+  { name: "Bob", surname: "Fish", age: 22 }
+]
+
+function map(arr, fn) {
+    let result = []
+
+    for (let index = 0; index < arr.length; index++) {
+        const element = arr[index]
+
+        if (element !== undefined) {
```

Comment:
no, you shouldn't validate `element`. Generally, the element may equal to `undefined`. The `map` method is not allowed to filter any values